ui/client/components/graph/node-modal/NodeDetailsContent.js

```diff
@@ -439,43 +445,55 @@ export class NodeDetailsContent extends React.Component {
     this.doCreateExpressionField(fieldName, fieldName, `${listFieldPath}.${expressionProperty}`, validators);
 
   doCreateExpressionField = (fieldName, fieldLabel, exprPath, validators) => {
```

Comment:
No you can do separeted component: `ExpressionField`. We can use it in future to refactor this huge NodeDetailsContent.

Reply:
Done